frontend/coding-challenge/src/app/shared/components/challenge/challenge.component.ts

```diff
@@ -250,11 +251,6 @@ export class ChallengeComponent implements OnInit {
         this.hideMsgFileUplod = false;
 
         element.setAttribute("style", "border-color:red;");
-      } else if (files[0].size > 52428800 && !files[0].name.includes('.zip')) {
```

Comment:
Don't remove this else-if-block, the fix of this block happens in another pull request